### PR TITLE
Sets terminateDebuggee to false

### PR DESCRIPTION
**Pull request by JohanMabille**
Stopping the debugger should not terminate the kernel. We noticed this bug when switching from PTVSD to debugpy in xeus-python. PTVSD does not take this argument into account (that's a bug), but debugpy does.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

A couple of tests are failing, which are related to the switch to `debugpy`, as well as this change to the protocol: https://github.com/jupyter-xeus/xeus-python/pull/314

Will push some commits to the branch directly.

File: packages/debugger/test/session.spec.ts [hunk 1]
```diff
@@ -73,7 +73,9 @@ describe('Debugger.Session', () => {
       });
       await debugSession.start();
       await debugSession.stop();
-      expect(events).toEqual(['output', 'initialized', 'process']);
+      expect(events).toEqual(
+        expect.arrayContaining(['output', 'initialized', 'process'])
+      );
```
Comment (jtpio) [inline packages/debugger/test/session.spec.ts:78]: This makes the tests a bit more flexible as other event messages might appear in this sequence.

File: packages/debugger/test/session.spec.ts [hunk 2]
```diff
@@ -228,8 +230,10 @@ describe('protocol', () => {
         frameId
       });
       const scopes = scopesReply.body.scopes;
-      expect(scopes.length).toEqual(1);
-      expect(scopes[0].name).toEqual('Locals');
+      expect(scopes.length).toBeGreaterThanOrEqual(1);
```
Comment (jtpio) [inline packages/debugger/test/session.spec.ts:233]: Something caught on the UI too: https://github.com/jupyterlab/jupyterlab/pull/9346

**Comment by jtpio** [comment]
Looks like the only left is the test for `loadedSources`, which now seems to time out, while we were expecting an error with `ptvsd`:

image

In https://github.com/jupyterlab/debugger/pull/170#discussion_r347252740 we decided to keep the test, even though the `loadedSources` request is not used in the UI. But could consider remove it now.

**Comment by JohanMabille** [comment]
I think it's fine skipping it for now. I'll open an issue in xeus-python, we'll have to to fix it at some point.

**Comment by jtpio** [comment]
Adding to 3.0 as this addresses the following user-facing issue:

xeus-python-restart

And fixes the tests.

**Comment by JohanMabille** [comment]
Actually the `loadedSources` request should not be sent to debuggers that don't support it (see [here](https://microsoft.github.io/debug-adapter-protocol/specification#Requests_LoadedSources)). This support can be checked in the reponse of the initialize request (and debugpy doe snot support it). So I think we should skip this test when the kernel is xeus-python.

**Comment by jtpio** [comment]
ok, thanks for checking :+1: 

Then let's remove the `loadedSources` test entirely for now.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**